models/stg_quickbooks__estimate.sql

```diff
@@ -45,6 +45,7 @@ final as (
         customer_id,
         cast(department_id as {{ dbt.type_string() }}) as department_id,
         cast( {{ dbt.date_trunc('day', 'due_date') }} as date) as due_date, 
+        exchange_rate,
```

Comment:
Please be sure to add this to the [src](https://github.com/fivetran/dbt_quickbooks_source/blob/f9fb58596c28c395a4d45ac1e42fb0f3e2d694b6/models/src_quickbooks.yml#L386-L411) and [stg.yml](https://github.com/fivetran/dbt_quickbooks_source/blob/f9fb58596c28c395a4d45ac1e42fb0f3e2d694b6/models/stg_quickbooks.yml#L489-L522) files and regen the docs.